1750998672579	Sync.LogManager	DEBUG	Flushing file log
1750998672580	Sync.LogManager	DEBUG	Log cleanup threshold time: 1750134672580
1750998672592	Sync.LogManager	DEBUG	Done deleting files.
1750998673603	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1751002186354	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1751002186354	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1751002186355	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1751002186357	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1751002188618	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1751002188618	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1751002188619	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1751002188620	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1751002188621	Sync.Service	DEBUG	User-Agent: Zen/1.13.2b (Linux x86_64) FxSync/1.141.0.20250616094514.desktop
1751002188621	Sync.Service	INFO	Starting sync at 2025-06-27 07:29:48 in browser session Y5RdNG-UgYvj
1751002188621	Sync.Service	DEBUG	In sync: should login.
1751002188621	Sync.Service	INFO	User logged in successfully - verifying login.
1751002188621	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1751002188621	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1751002188621	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1751002188621	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1751002188621	Sync.ErrorHandler	ERROR	Sync encountered a login error
1751002188621	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1751002188621	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1751002188622	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.sys.mjs:1051:15) JS Stack trace: onNotify@service.sys.mjs:1051:15
1751002188622	Sync.Service	DEBUG	Not syncing: login returned false.
